### PR TITLE
Some clarity to the .env.example file

### PR DESCRIPTION
**Pull request by mischat**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -13,19 +13,19 @@ BOSON_TOKEN=0x056b1ce5510ceefa6c33600551e55333aff8296f
 # DAI token Rinkeby address
```
Comment (zajck) [inline .env.example:13]: This should be without rinkeby.

Also line 10 `# local hardhat address #9` is not ok description.

**Review by zajck — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (mischat) [reply .env.example:13]: @zajck not sure what you mean, re: rinkeby ? what DAI token address is that ? which chain ?

And i can update line 10

Comment (zajck) [reply .env.example:13]: I've discussed that with @hswopeams 

The values are ok, it just wasn't clear for me, if the defaults have any meaning. And they might confuse someone when they see them first.

So I would just add comment like "Default values for BOSON_TOKEN and DAI_TOKEN are required for local deployment. To do deployment on chain, adjust them accordingly" (or something similar that tells the same)

Comment (mischat) [reply .env.example]: @zajck updated.

**Review by zajck — DISMISSED** [review]

File: .env.example [hunk 2]
```diff
@@ -7,25 +7,24 @@ CMC_API_KEY=000000000000000000000000000000000000000
...
-VOUCHERSETS_METADATA_URI=changeme
-VOUCHERS_METADATA_URI=changme
+CONDITIONAL_COMMIT_TOKEN_METADATA_URI=https://metadata.bosonportal.example/quest/{id}.json
+VOUCHERSETS_METADATA_URI=https://metadata.boson.example/voucherset/{id}.json
+VOUCHERS_METADATA_URI=https://metadata.boson.example/voucher/
 
 # Max tip in gwei - use "Fastest" setting from: https://www.etherchain.org/api/gasPriceOracle
-MAX_TIP = 13.1
\ No newline at end of file
+MAX_TIP = 13.1# API Keys
```
Comment (hswopeams) [inline .env.example:30]: why is # API Keys needed?

**Review by hswopeams — CHANGES_REQUESTED** [review]

File: .env.example (hunk 2 above)
Comment (mischat) [reply .env.example:30]: :)

Comment (mischat) [reply .env.example]: fixed ...

**Review by zajck — APPROVED** [review]

**Review by hswopeams — APPROVED** [review]

**Pull request merged**